### PR TITLE
Disc 361 origin cleanup

### PR DESCRIPTION
**Pull request by VictorHarbo**
Clean up clean up clean up

### CODE REVIEW

File: conf/ds-present-behaviour.yaml [hunk 1]
```diff
@@ -112,7 +112,7 @@ config:
 
   # Collections are to be set in ds-present-kb-collections-yaml if possible, as that file overwrites the collections defined here
   # The collections available for this instance of ds-present
-  #collections:
+  #origins:
```
Comment (tokee) [inline conf/ds-present-behaviour.yaml:115]: The comments should also be changed to use `origin`.

File: conf/ds-present-kb-collections.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ config:
                 - old_imageserver: ${path:config.oldimageserver.url}
 
   # The collections available for this instance of ds-present
-  collections:
+  origins:
```
Comment (tokee) [inline conf/ds-present-kb-collections.yaml:45]: The file itself (`ds-present-kb-collections.yaml`) should also be renamed.

File: src/main/java/dk/kb/present/StorageHandler.java [hunk 3]
```diff
@@ -14,22 +14,19 @@
  */
 package dk.kb.present;
 
+import dk.kb.present.storage.DSStorage;
 import dk.kb.present.storage.Storage;
 import dk.kb.present.storage.StorageController;
 import dk.kb.util.yaml.YAML;
-import javassist.NotFoundException;
```
Comment (tokee) [inline src/main/java/dk/kb/present/StorageHandler.java:20]: What was that and how did it ever work!?

File: src/main/java/dk/kb/present/StorageHandler.java [hunk 4]
```diff
@@ -14,22 +14,19 @@
...
-import javax.management.ServiceNotFoundException;
-import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 /**
  * Creates {@link DSStorage}s from a given configuration. Storages are used for retrieving raw records by
- * {@link DSCollection} and can be shared between multiple collections.
+ * {@link DSOrigin} and can be shared between multiple originss.
```
Comment (tokee) [inline src/main/java/dk/kb/present/StorageHandler.java:29]: `originss` → `origins`

File: src/main/java/dk/kb/present/api/v1/impl/IiifPresentationApiServiceImpl.java [hunk 5]
```diff
@@ -22,6 +22,7 @@
 import dk.kb.present.model.v1.ManifestServicesDto;
 import dk.kb.present.model.v1.ManifestStartDto;
 import dk.kb.present.model.v1.ManifestThumbnailDto;
+import dk.kb.present.model.v1.OriginDto;
```
Comment (tokee) [inline src/main/java/dk/kb/present/api/v1/impl/IiifPresentationApiServiceImpl.java:25]: Not used. But in general we have a bad habit of keeping unused imports.,,

**Review by tokee — CHANGES_REQUESTED** [review]
We thank the developers of IntelliJ IDEA for their skills at making refactoring work. Minor comments and you only missed two spots (that I could find):

* The `View`-constructor takes `originOfCollection`, which now is simply `origin`
* The Javadoc for the `stripPrefix` in `FileStorage`-constructor

File: src/main/java/dk/kb/present/StorageHandler.java (hunk 3 above)
Comment (VictorHarbo) [reply src/main/java/dk/kb/present/StorageHandler.java:20]: wauw! No clue!

File: conf/ds-present-behaviour.yaml (hunk 1 above)
Comment (tokee) [reply conf/ds-present-behaviour.yaml:115]: I took the liberty of fixing the remaining two old-name-in-comments.

**Pull request merged**